### PR TITLE
fix: voiceover and safari does not annouce selected value

### PR DESCRIPTION
**Pull request by wsuwt**
## Description

Move `role=combobox` to internal `div` that has the same level as `role=listbox`. This way Safari and VoiceOver will recognize the relationship between the two.  

- Added basic accessibility test cases
- Update documentation content and live examples
- Update demo page to be accessible

Fixes [ELF-2131](https://jira.refinitiv.com/browse/ELF-2131)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: packages/elements/src/select/index.ts [hunk 1]
```diff
@@ -1043,49 +994,73 @@ export class Select extends ControlElement implements MultiValue {
...
+  /**
+   * Area that allows select to be clickable and toggle between open and close
+   */
+  private get triggerTemplate(): TemplateResult {
+    return html` <div id="trigger" @tapstart="${this.toggleOpened}"></div>`;
   }
 
   /**
-   * A `TemplateResult` that will be used
-   * to render the updated internal template.
```
Comment (Sakchai-Refinitiv) [inline packages/elements/src/select/index.ts]: It should not be removed

File: packages/elements/src/select/index.ts
Comment (Sakchai-Refinitiv) [inline packages/elements/src/select/index.ts:1]: Please refactor the `eventOptions` for the scroll event to be the same v7.

File: packages/elements/src/select/index.ts (hunk 1 above)
Comment (wsuwt) [reply packages/elements/src/select/index.ts]: Updated.

File: packages/elements/src/select/index.ts
Comment (Sakchai-Refinitiv) [reply packages/elements/src/select/index.ts:1]: Updated

**Review by Sakchai-Refinitiv — APPROVED** [review]
Nice 👍

**Pull request merged**